src/mainviews/create-job.tsx

```diff
@@ -680,6 +680,9 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
             value={props.model.outputPath}
             id={`${formPrefix}outputPath`}
             name="outputPath"
+            helperText={trans.__(
+              'Relative path where output files are written'
```

Comment:
Believe a more helpful helper text would be: "Path to directory in which output files will be written. Path must be relative to the server root." @ellisonbg Thoughts?